### DISCUSSION TITLE
Rust Stdout/Stderr doesn't displayed on Android

### DISCUSSION THREAD

**Discussion by vincent-herlemont** [fzyzcjy/flutter_rust_bridge#671] [Q&A] [answered]
When I start my app for android `flutter run -d <my android emulator>`, no values from stdout/stderr from the rust code are displayed.

Is there something to do in particular to see the values returned by `println("")` or the traces returned by panic? 

NOTE: That works perfectly when I start my app for Linux: `flutter run -d linux`.

**Comment by fzyzcjy** [comment, marked as answer]
http://cjycode.com/flutter_rust_bridge/feature/stream.html?highlight=logging#example-logger-in-production

I will move this article into http://cjycode.com/flutter_rust_bridge/feature/logging.html in a minute.

**Reply by vincent-herlemont** [reply]
Thank you for your answer. I had seen the logger tutorial but in rare case of the Rust program panic (because of a dependency which use unwrap for example) how I can catch this and display or retrieve it on Android?

**Reply by fzyzcjy** [reply]
http://cjycode.com/flutter_rust_bridge/feature/handler.html

“[Example: Report errors to your backend in addition to telling Dart](http://cjycode.com/flutter_rust_bridge/feature/handler.html#example-report-errors-to-your-backend-in-addition-to-telling-dart)”
